### ISSUE TITLE
noone currently can cancel / close the sub-window opened by e.g. " clone repository ... " from file menu action

### ISSUE THREAD

**Issue by arkenidar** [sublimehq/sublime_merge#1989]
**Problem description**

no-one currently can cancel / close the sub-window opened by e.g. " clone repository ... " from file menu action .
same on windows 11 and debian 12 .

**Preferred solution**

I would advice to offer some way to cancel and close such " panel " , e.g. I changed my mind to do other actions and this "panel" gets in the way and closing is needed . to close the whole app and re-open it ... not an elegant way , it seems to me .

**Alternatives**

adding some way to close just this superimposed "panel" is enhancement to closing the whole app .

**Screenshots**
![image](https://github.com/user-attachments/assets/7d961a13-c837-4271-9ffe-968096c1322d)
this panel / sub-window cannot be closed e.g. if I don't want to do anything of the offered options for action i.e. to cancel .

**Comment by RobertPaauw** [comment]
I agree this is not super intuitive, but does closing _new tab_ work for you?

Maybe an extra "close" button on the panel itself (that closes the tab) would have been more intuitive here (but less minimalistic).

![close_the_tab](https://github.com/user-attachments/assets/63f3dbe8-1c52-4e96-9a3f-2ddbda874398)

**Comment by arkenidar** [comment]
ah ! now ... I can relate : it is not superimposed , it is on a tab of its own .
thanks . that is enough for me . now I know and understand . sorry .

furthermore : what about covering all the rest in background ( misleading to me , but possibly others ) in order to highlight this " current logic " of " not a covering window but a tab of its own " ?

design choices maybe legacy of what is already there . I can relate possibly to some of that .
thanks again and sorry again .
